<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Maintenance</class>
 <widget class="QWidget" name="Maintenance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>312</width>
     <height>231</height>
    </rect>
   </property>
   <property name="gridVisible">
    <bool>true</bool>
   </property>
   <property name="navigationBarVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableView" name="startDateTable">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QLabel" name="startDateLabel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Start date of:</string>
   </property>
  </widget>
  <widget class="QTableView" name="deadlineTable">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>150</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QLabel" name="deadlineLabel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>130</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Deadline of:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtInfo">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>30</y>
     <width>151</width>
     <height>151</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="informationLabel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Information</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addBtn">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>190</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Assign</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeBtn">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>190</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
